pkg/apis/addtoscheme_nmstate_v1.go

```diff
@@ -0,0 +1,10 @@
+package apis
+
+import (
+	v1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes, v1.SchemeBuilder.AddToScheme)
```

Comment:
It looks like you kept types.go we had there originally, right?

operator-sdk has a lot of metadata, could you remove all we had for now, and then add NodeNetworkState using operator-sdk

`operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService`